src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+    private final FigureType figureType = FigureType.RECTANGLE;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public int getFirstSide() {
+        return firstSide;
+    }
+
+    public int getSecondSide() {
+        return secondSide;
+    }
+
+    public FigureType getFigureType() {
+        return figureType;
+    }
+
+    @Override
+    public int getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getFigureType()
+                + ", area: " + getArea()
+                + " sq.units, firstSide: " + getFirstSide()
+                + " units, secondSide: " + getSecondSide()
```

Comment:
```suggestion
                + " sq.units, firstSide: " + firstSide
                + " units, secondSide: " + secondSide
```
if we can access fields directly - it looks better if we address them directly